### DISCUSSION TITLE
Error on cache.validator clear

### DISCUSSION THREAD

**Discussion by oolja** [symfony/symfony#50731] [Q&A]
Hello,
I get this error when I try to clear the cache pool:
`The "cache.validator" service or alias has been removed or inlined when the container was compiled. You should either make it public, or stop using the container directly and use dependency injection  
   instead.`
Any idea how to fix this?

**Comment by ghost** [comment]
What do you mean by ` I try to clear the cache pool`? `bin/console ca:cl` command?
I don't know. More context needed. Personally I never waste time on errors from `ca:cl` and do `rm -rf var/cache/*`. If then an error occurs then I start to worry.

**Reply by oolja** [reply]
I run: `bin/console cache:pool:clear --all`

**Reply by ghost** [reply]
Please fill in your discussion with Symfony version you use and the exception stack trace. Also, is it a problem that came up unexpected or did you do something special with your app like for example deps upgrade, config change etc?

Always remember to provide as many details as possible because saying "I've got an error" is not much.
